ui/job-view/headerbars/FiltersMenu.jsx

```diff
@@ -3,116 +3,84 @@ import PropTypes from 'prop-types';
 
 import { thAllResultStatuses } from '../../js/constants';
 
-export default class FiltersMenu extends React.Component {
-  constructor(props) {
-    super(props);
+const resultStatuses = thAllResultStatuses.filter(rs => rs !== 'runnable');
 
-    this.resultStatuses = thAllResultStatuses.slice();
-    this.resultStatuses.splice(thAllResultStatuses.indexOf('runnable'), 1);
+export default function FiltersMenu(props) {
```

Comment:
This is another case that looks worse than it is.  I was able to make this a stateless function as well.  Functionality didn't change here at all.  Sorry the github diff looks terrible...